src/IconButton/IconButton.scss

```
@@ -0,0 +1,41 @@
+$iconbutton-hover-size: 1.6em;
```

```suggestion
$iconbutton-hover-size: 1.6em !default;
```

Add a default here in case a consumer of paragon would change this in a theme

---

Oh, cool, I didn't know about `!default`. :)